test/parallel/test-domain-abort-when-disposed.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const common = require('../common');
+const JSStream = process.binding('js_stream').JSStream;
```

Comment:
Sorry, not sure what you mean by "put this together with `Socket`". Mind clarifying?